src/core/lib/iomgr/tcp_server_posix.c

```diff
@@ -114,6 +114,8 @@ struct grpc_tcp_server {
 
   /* is this server shutting down? */
   bool shutdown;
+  /* have listeners been shutdown? */
+  bool shutdown_listeners;
```

Comment:
We do not have a memset on the server creation. I guess a s->shutdown_listerners = false; is needed in the tcp_server_create?